### ISSUE TITLE
Populating the map list for the create server window is blocking

### ISSUE THREAD

**Issue by illwieckz** [Unvanquished/Unvanquished#1157]
That's a bit similar to: https://github.com/DaemonEngine/Daemon/issues/284 _“Server list refresh is blocking”_
I don't know if it's an engine thing or a cgame thing.

**Comment by DolceTriade** [comment]
There are some blocking calls for DNS resolution and stuff that can't be changed unless with fetch it on a separate thread and check for completion asynchronously.

**Comment by illwieckz** [comment]
Yep, without a working network, querying master servers is blocking until dns query timeout.
